### PR TITLE
Chore(md): update readme + adding contribution guideline

### PR DESCRIPTION
**Pull request by AugustinMauroy**
## Readme

* Remove content from nextjs
* Add logo

## Contribution guideline

* Add commit name convention
* Add pr title convention

The goal of contribution guideline it's to keep project healthy. And to And to make the project open to everyone

### CODE REVIEW

**Review by chrisgervang — COMMENTED** [review]
Can you please explain the difference in the two logos? "withe" is likely a typo in the title.

We can probably call them "logo-color.svg" and "logo-black.svg". Please update the logo references in the website as well.

**Comment by AugustinMauroy** [comment]
@chrisgervang true I will update (in website).
I use black because the text is in dark and I use white because the text is in withe

**Comment by AugustinMauroy** [comment]
Yup I don't found any code about usage of logo

**Comment by chrisgervang** [comment]
I took a look and, and I see what you mean. The [logo](https://github.com/openjs-foundation/open-visualization/blob/main/components/hero.tsx#L58) is used but not the color logo. Please just rename the [logo-withe-color.svg](https://github.com/openjs-foundation/open-visualization/pull/4/files#diff-fff2b9d3bfe378ff6a49c6ff5281a611687c2dde656c11e1252a02a53e8cf23d) file to `logo-white-color.svg`

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Contribution guideline
```
Comment (chrisgervang) [inline CONTRIBUTING.md:1]: @bensternthal does OpenJS have a standard contribution guide we should incorporate into this?

**Comment by AugustinMauroy** [comment]
About logo we should also put on artwork

File: CONTRIBUTING.md (hunk 1 above)
Comment (bensternthal) [reply CONTRIBUTING.md:1]: Not that I am aware of. I think most projects customize the contribution guidelines to the specific needs of the project.

You can see the full list of contribution guides [here](https://github.com/openjs-foundation/cross-project-council).

**Comment by AugustinMauroy** [comment]
close because it's useless

**Pull request closed without merging**